src/Pact/Repl.hs

```diff
@@ -132,7 +132,7 @@ initEvalEnv ls = do
   mv <- newMVar ls
   return $ EvalEnv (RefStore nativeDefs) mempty Null Transactional
     def def mv repldb def pactInitialHash freeGasEnv
-    permissiveNamespacePolicy (spvs mv) def def def
+    permissiveNamespacePolicy (spvs mv) def def def True
```

Comment:
what's this change?

Reply:
it's the new flag for the `EvalEnv`. I can make this look a lot better actually.